README.md

```diff
@@ -1,11 +1,20 @@
-# multiboot2-elf64
+# Rust crate: multiboot2
 ![Build](https://github.com/rust-osdev/multiboot2-elf64/actions/workflows/rust.yml/badge.svg)
 [![crates.io](https://img.shields.io/crates/v/multiboot2.svg)](https://crates.io/crates/multiboot2)
 [![docs](https://docs.rs/multiboot2/badge.svg)](https://docs.rs/multiboot2/)
 
-An experimental Multiboot 2 crate for ELF-64 kernels. It's still incomplete, so please open an issue if you're missing some functionality. Contributions welcome!
+A crate for parsing multiboot2 structures and relevant multiboot2 type definitions. The primary goal is to parse
+the tags from the "multiboot2 information structure" (also called *mbi*). This is the data structure passed 
+via `ebx` register (on `x86`) when the bootloader gives control to the payload (for example your kernel).
+Furthermore, it contains type definitions for parsing a "multiboot2 header" including the "multiboot2 header tags".
 
-It uses the Multiboot 2.0 specification at https://www.gnu.org/software/grub/manual/multiboot2/multiboot.html and the ELF 64 specification at http://www.uclibc.org/docs/elf-64-gen.pdf.
+It's still incomplete, so please open an issue if you're missing some functionality. Contributions welcome!
+It uses the Multiboot 2.0 specification at https://www.gnu.org/software/grub/manual/multiboot2/multiboot.html and the 
+ELF 64 specification at http://www.uclibc.org/docs/elf-64-gen.pdf.
+
+⚠ The `elf64` in the github repository url is a bit misleading. It means the crate can parse the `elf`-sections 
```

Comment:
I probably agree with this, but rather than add this comment, I think it would be better to rename the repository. The crate is already just called `multiboot2`

cc @rust-osdev/multiboot2 does anyone feel that we shouldn't do that?